src/components/Trading/TradePropertiesForm.tsx

```diff
@@ -98,7 +98,8 @@ function TradePropertiesForm(props: TradePropertiesFormProps) {
   const [manualPrice, setManualPrice] = React.useState<ManualPrice>({})
   const [priceMode, setPriceMode] = React.useState<"fixed-buying" | "fixed-selling">("fixed-selling")
 
-  const defaultPrice = bigNumberToInputValue(priceMode === "fixed-buying" ? BigNumber(1).div(props.price) : props.price)
+  const price = props.price.eq(0) ? BigNumber(1) : props.price // prevent division by zero
+  const defaultPrice = bigNumberToInputValue(priceMode === "fixed-buying" ? BigNumber(1).div(price) : price)
```

Comment:
```suggestion
  const inversePrice = props.price.eq(0) ? BigNumber(0) : BigNumber(1).div(price) // prevent division by zero
  const defaultPrice = bigNumberToInputValue(priceMode === "fixed-buying" ? inversePrice : price)
```

Reply:
Looks good,
I was a bit unhappy with my naming of "price" anyways